### DISCUSSION TITLE
[Mobile] Difficult and can't do remove or apply style in link text

### DISCUSSION THREAD

**Discussion by ghost** [singerdmx/flutter-quill#672] [Mobile]
My issue is about [Mobile]

Link attribute text is difficult and can't apply or remove style by selecting. When I select on link text, it appears bottom menu bar: open, copy and remove. So, I can't apply style or remove

How can I apply or remove style by selecting link text.

Thank you very much

**Comment by xinyuezhang0402** [comment]
Currently not supporting.
But you can submit PR for it

**Reply by ghost** [reply]
Thank you very much for your quick response and answering

How can I submit feature request for it? @xinyuezhang0402

**Reply by xinyuezhang0402** [reply]
First write code and when you are ready please let us know
